### ISSUE TITLE
gem install thymeleaf produces error on Ruby 2.4.1p111

### ISSUE THREAD

**Issue by dhughes** [trabe/thymeleaf-rb#4]
I am running Ruby 2.4.1 as shown here:

```
$ ruby -v
ruby 2.4.1p111 (2017-03-22 revision 58053) [x86_64-darwin16]
```

When I try to install Thymeleaf-rb using `gem install` I receive this output:

```
$  gem install thymeleaf
ERROR:  Error installing thymeleaf:
	thymeleaf requires Ruby version ~> 2.3.0.
```

I am somewhat new to Ruby, but I'm guessing that version _only_ 2.3.0 is supported, as per [line 10 in thymeleaf.gemspec.](https://github.com/trabe/thymeleaf-rb/blob/master/thymeleaf.gemspec#L10)

Is this intentional? Is this gem known not to work on newer versions of Ruby? If not, then perhaps newer versions of ruby could be supported?

**Comment by davidbarral** [comment]
The gem should also work with Ruby 2.4. I guess the restriction was there to force us to test it against the newer Ruby releases. We'll fix the Travis config ASAP to test the gem and publish a new release.

**Comment by davidbarral** [comment]
Just published a 0.1.3 version with the gemspec fixed. Please test the new version and confirm that everything works :)
